test/finance/test_european_call_expected_value.py

```diff
@@ -34,6 +35,11 @@ def setUp(self):
         super().setUp()
         self.seed = 457
         aqua_globals.random_seed = self.seed
+        warnings.filterwarnings(action="ignore", category=DeprecationWarning)
```

Comment:
Is this because of the move of some components to Terra where they are already marked deprecated in Aqua but since things have not been merged it has not been changed. Maybe in this case we should leave it emitting the deprecation warning since we should be fixing this test right?